go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/VictoriaMetrics/victorialogs-datasource
 go 1.24.1
 
 require (
+	github.com/VictoriaMetrics/VictoriaMetrics v1.113.0
```

Comment:
Please run `go mod tidy && go mod vendor` to remove dependency that you don't use.

Reply:
updated dependecies